ha/core/event_analyzer/parser/parser.py

```diff
@@ -168,8 +170,33 @@ def __init__(self):
         self.cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
         self.site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
         self.rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+
+        _index = 'cortx'
+        ConfigManager._safe_load(_index, "yaml:///etc/cortx/cluster.conf")
```

Comment:
cluster.conf should not be used anywhere except in mini privisioning. We should be using gconf search APIs